docker-compose/readme.md

```diff
@@ -452,22 +452,35 @@ You can run selenium tests by following below steps:
    ./docker-compose/run_selenium_tests_local.sh slsx
    ```
 
-3. To debug tests (visualize webUI interaction): point VNC client to localhost:5900
+3. To debug tests (visualize browser interaction): point VNC client to localhost:5900
 
    1. requires installation of vnc viewer, password (secret)
-   2. Start tests using -debug parameter as shown below:
-      use MSLSX (default)
+   2. Start tests using -d option as shown below:
+
+   use MSLSX (default)
 
    ```
-   ./docker-compose/run_selenium_tests_local.sh debug
+   ./docker-compose/run_selenium_tests_local.sh -d
    ```
 
    ```
-   ./docker-compose/run_selenium_tests_local.sh mslsx-debug
+   ./docker-compose/run_selenium_tests_local.sh -d mslsx
    ```
 
    use SLSX
 
    ```
-   ./docker-compose/run_selenium_tests_local.sh slsx-debug
+   ./docker-compose/run_selenium_tests_local.sh -d slsx
    ```
+
+## Running Selenium tests on remote BB2 server (SBX, TEST, PROD)
+
+You can run selenium tests against a remote bb2 server by following below steps:
+
+1. From the base directory of the local repo run:
+2. ./docker-compose/run_selenium_tests_remote.sh SBX (run selenium tests against bb2 server on SBX)
```

Comment:
@JFU-GIT To run following the instruction, the executable bit needs set on the file.

I think you can do this via the following code or similar:
```
chmod u+x ./docker-compose/run_selenium_tests_remote.sh
git add --chmod=+x ./docker-compose/run_selenium_tests_remote.sh
```